src/chrome/content/rules/DistributedWeaknessFiling.xml

```diff
@@ -0,0 +1,15 @@
+<ruleset name="Distributed Weakness Filing">
+
+	<target host="distributedweaknessfiling.org" />
+	<target host="*.distributedweaknessfiling.org" />

```

Comment:
we discourage the use of wildcard as a general styling practice, please see [/CONTRIBUTING.md#ruleset-style-guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#ruleset-style-guide) for details

Reply:
@gloomy-ghost But I think wildcard is better in this case, because `https://distributedweaknessfiling.org` sends a HSTS header with `includeSubDomains`, and I find it's hard to enumerate all its subdomains.

Reply:
ah okay, in that case can you add a comment in the ruleset?

Reply:
Done.